### PR TITLE
fix(wikipedia): misc

### PR DESCRIPTION
**Pull request by jh-devv**
## 🔧 What does this fix? 🔧

Fixes:
- Notification menu colors
- Loading bar colors
- Added colors for some new SVG icons

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/wikipedia/catppuccin.user.css [hunk 1]
```diff
@@ -261,11 +268,17 @@
     }
 
     .oo-ui-textInputWidget,
-    .oo-ui-inputWidget-input {
+    .oo-ui-inputWidget-input{
```
Comment (jh-devv) [inline styles/wikipedia/catppuccin.user.css:271]: Revert to
```suggestion
    .oo-ui-inputWidget-input {
```

**Comment by GitMuslim** [comment]
@uncenter why was this drafted?

**Review by GitMuslim — APPROVED** [review]

**Comment by GitMuslim** [comment]
oh wait woops

**Comment by jh-devv** [comment]
I'd say this is good to merge, though needs linting.

Still, this `Wikipedia userstyle` does not work on other `Wikipedia instances` than `en` even without this PR.
Since, some of these use the older `legacy` interface and have some custom objects in their pages.

image

I also want to say this userstyle has an incompatibly with that `dark-mode plugin` in the `Wikipedia settings`?

Though, the idea of this PR was to only fix some small things in the English Wikipedia for for example, notifications. I haven't merged it yet since I though of adding the other stuff but thats probably too much for this PR?

So yea, can merge!

**Pull request merged**